supernovacontroller/sequential/supernova_device.py

```diff
@@ -24,30 +25,31 @@ def id_gen(start=0):
 
 class SupernovaDevice:
     def __init__(self, start_id=0):
-      self.controller = TransferController(id_gen(start_id))
-      self.response_queue = queue.SimpleQueue()
-      self.notification_queue = queue.SimpleQueue()
-      self.notification_handlers = {}
+        self.controller = TransferController(id_gen(start_id))
+        self.response_queue = queue.SimpleQueue()
+        self.notification_queue = queue.SimpleQueue()
+        self.notification_handlers = {}
 
-      self.process_response_thread = threading.Thread(target=self._pull_sdk_response, daemon=True)
-      self.process_notifications_thread = threading.Thread(target=self._pull_sdk_notification, daemon=True)
+        self.process_response_thread = threading.Thread(target=self._pull_sdk_response, daemon=True)
+        self.process_notifications_thread = threading.Thread(target=self._pull_sdk_notification, daemon=True)
 
-      self.running = True
+        self.running = True
 
-      self.process_response_thread.start()
-      self.process_notifications_thread.start()
+        self.process_response_thread.start()
+        self.process_notifications_thread.start()
 
-      self.driver = Supernova()
+        self.driver = Supernova()
 
-      self.interfaces = {
-          "i2c": [None, SupernovaI2CBlockingInterface],
-          "i3c.controller": [None, SupernovaI3CBlockingInterface],
-          "uart":[None, SupernovaUARTBlockingInterface],
-          "i3c.target": [None, SupernovaI3CTargetBlockingInterface],
-          "spi.controller": [None, SupernovaSPIControllerBlockingInterface],
-      }
+        self.interfaces = {
+            "i2c": [None, SupernovaI2CBlockingInterface],
+            "i3c.controller": [None, SupernovaI3CBlockingInterface],
+            "uart": [None, SupernovaUARTBlockingInterface],
+            "i3c.target": [None, SupernovaI3CTargetBlockingInterface],
+            "spi.controller": [None, SupernovaSPIControllerBlockingInterface],
+            "gpio": [None, SupernovaGPIOInterface],
+        }
 
-      self.mounted = False
+        self.mounted = False
```

Comment:
Are these added spaces neccessary?

Reply:
YES. As you can check, we are correcting the inconsistent indentation that was present there. Originally there was 2 spaces for the indentation, whereas in the rest of the code is 4. So, we update that part to match the rest of the code.